packages/protocol/contracts/locking/IStakingLocking.sol

```diff
@@ -1,31 +1,38 @@
 pragma solidity ^0.5.17;
 
 
+// Note: seems weird that this is in the locking/ dir; perhaps we should move it up to be the same dir as Staking?
+// Note: perhaps we can find a better name... like StakingWithLockManagement?
 interface IStakingLocking {
     event NewLockManager(address indexed account, address indexed lockManager, bytes data);
     event Unlocked(address indexed account, address indexed lockManager, uint256 amount);
+    // Note: the increase param for these two does not seem that useful
```

Comment:
Why not? How do you know then if the `_amount` was added or subtracted?

Reply:
Yes, that's true, and I didn't explain myself very well here. I was thinking that perhaps we should just emit the current total, as opposed to the increase/decrease amount, similar to how the `Approval` event works in ERC20s.